test/integration/ynEIGEN/ynEigen.t.sol

```diff
@@ -589,4 +666,6 @@ contract ynEigenDonationsTest is ynEigenIntegrationBaseTest {
 
         assertGt(bobShares, 1 wei, "Bob's shares should be greater than 1 wei");
     }   
-}
\ No newline at end of file
+}
+
+// | src/ynEIGEN/ynEigen.sol                                 | 85.25% (52/61)    | 87.67% (64/73)     | 100.00% (20/20)  | 75.00% (12/16)
```

Comment:
it's good you measured this but let's not put it in the file, just leave it in the description of the file.

It becomes a chore to keep updating it as we add code or add tests.

Reply:
https://github.com/yieldnest/yieldnest-protocol/pull/135/commits/e2329a727a9699c7e18b38915014c3dec67cbf91